crates/miden-lib/asm/note_scripts/P2ID.masm

```diff
@@ -29,7 +29,7 @@ const.ERR_P2ID_TARGET_ACCT_MISMATCH="P2ID's target account address and transacti
 #!   greater than 2^63.
 begin
     # store the note inputs to memory starting at address 0
-    padw push.0 exec.note::get_inputs
+    padw push.0 exec.active_note::get_inputs
```

Comment:
```suggestion
    padw push.0 exec.executing_note::get_inputs
```

Not specific to this line, but I'm not quite sold on the "active note" terminology. It's fine with an explanation, but I think "executing note" might be more self-explanatory in the Miden context and when reading a line like this in a note script. This might need some discussion, cc @bobbinth or @partylikeits1983. Wdyt?